test/unit/component/parser/json/BpmnJsonParser.sequenceFlow.conditional.test.ts

```diff
@@ -0,0 +1,134 @@
+/**
```

Comment:
:warning: we should add a comment in `ShapeBpmnElementKind` to warn contributors about the need of uncommenting tests when adding new supported elements (for instance, manualTask)

Reply:
I forgot it.
Done